.github/workflows/build-and-test.yaml

```diff
@@ -50,10 +71,22 @@ jobs:
         minor="$(echo ${src_version} | cut -d. -f2)"
         patch="$(echo ${src_version} | cut -d. -f3 | cut -d- -f1)"
 
-        effective_version=${major}.${minor}.${patch}-${commit}
+        if ${{ inputs.release || false }}; then
```

Comment:
ditto (`|| false`)?

Reply:
see above. I have yet to find a more elegant approach. one option might be to pull this logic into the `params` step, so we at least onyl have to do this once, wdyt?